assets/tests/setup.tsx

```diff
@@ -11,11 +11,13 @@ jest.mock("react-leaflet-fullscreen", () => ({
   default: jest.fn(() => null),
 }))
 
-jest.mock("react-tooltip", () => ({
+jest.mock("@tippyjs/react", () => ({
   __esModule: true,
   default: jest
     .fn()
-    .mockImplementation(() => <div className="mock-react-tooltip" />),
+    .mockImplementation((props) => (
+      <div className="mock-tippy">{props.children}</div>
```

Comment:
One thing that I didn't consider when suggesting this approach is that, since react-tooltip used the various `data-*` attributes on other elements, the contents of the tooltips themselves made it into the test snapshots that way. To make sure that they still make it into the snapshots with Tippy, I think you can change this to something like:
```
jest.mock("@tippyjs/react", () => ({
  __esModule: true,
  default: jest.fn().mockImplementation((props) => (
    <div className="mock-tippy">
      <div className="mock-tippy-content">{props.content}</div>
      {props.children}
    </div>
  )),
}))
```
and then rebuild the snapshots.